mods/marytts/test/tts.unit.test.ts

```diff
@@ -10,52 +28,45 @@ import fs from "fs";
 const expect = chai.expect;
 chai.use(sinonChai);
 chai.use(chaiAsPromised);
+chai.should();
 const sandbox = sinon.createSandbox();
 
-if (process.env.NODE_ENV === "dev") {
-  require("dotenv").config({path: path.join(__dirname, "..", "..", ".env")});
-}
-
 describe("@fonos/marytts", () => {
   afterEach(() => sandbox.restore());
+  const defConfig = {
+    host: "localhost",
+    port: 59125,
+    locale: "EN_US"
+  };
+
+  sandbox.stub(MaryTTS.prototype, "init").returns();
 
-  it("rejects if the TTS engine response is not 200", () => {
+  it("Should rejects if the TTS engine response is not 200", async () => {
     const join = sandbox.spy(path, "join");
     const createWriteStream = sandbox.spy(fs, "createWriteStream");
-    const get = sandbox.stub(http, "get");
     const pipe = sandbox.stub();
-    const response = {
-      statusCode: 201,
-      pipe
-    };
-    get.yields(response);
-
-    const tts = new MaryTTS();
+    const get = sandbox.stub(http, "get").yields({statusCode: 201, pipe});
 
-    expect(tts.synthesize("hello world")).to.eventually.rejectedWith(
-      "Request failed status code"
-    );
+    const tts = new MaryTTS(defConfig);
 
+    await expect(tts.synthesize("hello world", {locale: "en_US"}))
+      .to.be.eventually.rejected.and.to.be.an.instanceOf(Error)
+      .to.have.property("message", "Request failed status code: 201");
     expect(pipe).to.not.have.been.called;
```

Comment:
Be specific on how many times should they be called `pipe` and `createWriteStream`